src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2241,14 +2261,32 @@ public static Map<String,String> getInvertiblePinMap(Series series, Unisim unisi
 	 * instances from which to copy the implementation. If targeting the top instance, use an 
 	 * empty String ("") as the destination instance name.
 	 */
-	public static void copyImplementation(Design src, Design dest, boolean lockPlacement, 
+	public static void copyImplementation(Design src, Design dest, boolean lockPlacement,
+										  boolean lockRouting, Map<String,String> srcToDestInstNames) {
+		copyImplementation(src, dest, false, false, lockPlacement, lockRouting, srcToDestInstNames);
```

Comment:
Are `copyStaticNets=false` and `copyOnlyInternalNets=false` good defaults?

Reply:
I believe it depends on a use case.  see above comment.

Reply:
I would say it's okay to break older under-featured functionality in favour of more working features. It seems reasonable to expect static nets to be copied, and to not copy them be the exception (which users can explicitly set with the overload).

What's the difference between the two values for `copyOnlyInternalNets`? Why was the old functionality `false` -- meaning it would copy a net's sinks (if it was internal) but not the source since that's external?

Reply:
Based on previous conversation with Chris, it is always expected that Vivado will be invoked after copyImplementation. Thus, I would consider my use case an outliner that need to preserve the static routes. Besides, we can defer this decision until when we need to make it.

Reply:
@clavin-xlnx, do you have comment about eddie question?

Reply:
Apart from my reason above, I'm not confident to make copy static nets a default for one technical reason. Copying static nets requires preserving their sources. These sources could be outside of the placement footprint of other cells. I can imagine some users do not want those location blocked, but want to merge the static sources with those of other cells.  

It is true that such users can turn off copying static nets, if that a default. There is no clear winner and thus logistically, we should not hold up this PR and defer the decision until the full impact is vetted.

Reply:
Leave as is.